### DISCUSSION TITLE
GraphQL: Is there a mutation to transfer an issue to a different repo?

### DISCUSSION THREAD

**Discussion by ynie** [community/community#24868] [Apps, API and Webhooks]
<p>Hello all,</p>
<p>I’m looking for the GraphQL mutation to transfer an issue to a different repo? If not, is there an old API for it? Thanks!</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/ynie">@ynie</a>!</p>
<p>This should be the <a href="https://developer.github.com/v4/mutation/transferissue/" rel="nofollow noopener">GraphQL mutation for transferring issues</a> that you need. </p>
<p>Hope that helps!</p>

**Comment by ynie** [comment]
<p>Awesome! Thank you!</p>
